cluster_test.go

```diff
@@ -83,6 +136,8 @@ func TestInvalidHosts(t *testing.T) {
 		"foo://127.0.0.1:3000",
 	} {
 		_, err := newCluster(http.DefaultClient, invalidHost)
-		assert.Equal(t, err, ErrInvalidEndpoint, "undetected invalid host: %s", invalidHost)
+		if err == nil {
```

Comment:
`assert.Error()` would also work.

(Note: I'm not a huge fan of `stretchr/testify` but think we should be consistent with the existing assertion style where possible.)